apps/web/src/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationCard.tsx

```diff
@@ -41,92 +45,113 @@ const ApplicationCard = ({
+          level={headingLevel}
+          size="h6"
+          data-h2-margin="base(0, 0, x1, 0)"
+          data-h2-flex-grow="base(1)"
+        >
+          {application.poolAdvertisement
+            ? getFullPoolAdvertisementTitleHtml(
+                intl,
+                application.poolAdvertisement,
+              )
+            : ""}
+        </Heading>
+        <p data-h2-font-size="base(0.8rem)" data-h2-text-align="base(right)">
+          {intl.formatMessage(
+            {
+              defaultMessage: "ID: {id}",
+              id: "hEXXG0",
+              description: "Label for application ID",
+            },
+            { id: application.id },
+          )}
+        </p>
+      </div>
+      <p data-h2-margin="base(x1, 0)">
+        {applicationIsDraft
+          ? intl.formatMessage(
               {
-                defaultMessage: "ID: {id}",
-                id: "hEXXG0",
-                description: "Label for application ID",
+                defaultMessage:
+                  "Apply by: <strong><red>{closingDate}</red></strong>",
+                description:
+                  "Text notifying user of closing date for an application",
+                id: "eqVWC6",
               },
+              { closingDate },
+            )
+          : intl.formatMessage(
               {
-                id: application.id,
+                defaultMessage: "Applied on: <strong>{submittedAt}</strong>",
+                description:
+                  "Text notifying user of closing date for an application",
+                id: "NbOOsf",
               },
+              { submittedAt },
             )}
-          </p>
+      </p>
+      <div
+        data-h2-display="base(flex)"
+        data-h2-align-items="base(center)"
+        data-h2-gap="base(x1)"
+        data-h2-justify-content="base(space-between)"
+        data-h2-margin="base(x1, 0, 0, 0)"
+      >
+        <div
+          data-h2-display="base(flex)"
+          data-h2-align-items="base(center)"
+          data-h2-gap="base(x1)"
+        >
+          <ApplicationActions.SeeAdvertisementAction
+            show={notEmpty(application.poolAdvertisement)}
+            advertisement={application.poolAdvertisement}
+          />
+          <ApplicationActions.ViewAction
+            show={!applicationIsDraft}
+            application={application}
+          />
+          <ApplicationActions.SupportAction
+            show={!recruitmentIsExpired && !isApplicantPlaced}
+          />
+          <ApplicationActions.DeleteAction
+            onDelete={onDelete}
+            show={applicationCanBeDeleted}
+            application={application}
+          />
+          <ApplicationActions.ArchiveAction
+            onArchive={onArchive}
+            show={applicationCanBeArchived}
+            application={application}
+          />
         </div>
+        <ApplicationActions.ContinueAction
+          show={applicationIsDraft}
+          application={application}
+        />
```

Comment:
I think it should be like
`show={applicationIsDraft && !recruitmentIsExpired}`

Reply:
Thanks, added in [hide continue application if expired](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5744/commits/62b159ae85e113a05c244a3c742071c1fee8f6f6)